default/AI/ShipDesignAI.py

```diff
@@ -1445,7 +1404,7 @@ def _rating_function(self):
         if self.troops == 0:
             return INVALID_DESIGN_RATING
         else:
-            return self.troops/(self.production_cost**((1+foAI.foAIstate.shipCount * AIDependencies.SHIP_UPKEEP)**-1))
+            return self.troops/self._adjusted_production_cost()
```

Comment:
Ah, yes, it looks like I had done the spacing poorly -- for the terms in parenthesis, the '+' could optionally have space to both sides of it, but if it doesn't then the '*' should not.  Was there anything more to your comment Cj?